src/scss/main.scss

```diff
@@ -163,7 +163,6 @@ ul.meta {
   }
 
   .tag {
-    text-transform: capitalize;
```

Comment:
I believe I opted to create a `capitalize` 11ty filter instead, because we had some un-wanted results when using the CSS transform

e.g. compare https://templates.netlify.com/tags/netlifyCMS to https://deploy-preview-191--templates.netlify.app/tags/netlifycms where the tag is now correctly capitalized

Reply:
It's been a while though, so there could also have been some other factors